tests/cursor/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -0,0 +1,245 @@
+
+`;
+
+exports[`cursor-6.js 1`] = `
+const      y    /* h<|>i */   = 5
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+const y /* h<|>i */ = 5;
+
+`;
+
+exports[`cursor-7.js 1`] = `
+const y = 5
+<|>
+
+const z = 9
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+const y = 5;
+<|>
+const z = 9;
+
+`;
+
+exports[`cursor-8.js 1`] = `
+  func<|>tion banana(){}~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+func<|>tion banana() {}
+
+`;
+
+exports[`cursor-9.js 1`] = `
+     thisWillBeFormatted  <|>  (2  ,3,   )~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+thisWillBeFormatted<|>(2, 3);
+
+`;
+
+exports[`cursor-10.js 1`] = `
+const y = 5
+
+
+
+
+<|>
+  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+const y = 5;
+<|>
+`;
+
+exports[`file-start-with-comment-1.js 1`] = `
+// hi<|> lol
+haha()
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+// hi<|> lol
+haha();
+
+`;
+
+exports[`file-start-with-comment-2.js 1`] = `
+// hi lol
+haha()<|>
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+// hi lol
+haha()<|>;
+
+`;
+
+exports[`range-0.js 1`] = `
+thisWontBeFormatted  ( 1  ,3)
+
+    thisWillBeFormatted  <|>  (2  ,3,   )
+
+    thisWontBeFormatted  (2, 90  ,)
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+thisWontBeFormatted  ( 1  ,3)
+
+    thisWillBeFormatted<|>(2, 3);
+
+    thisWontBeFormatted  (2, 90  ,)
+    
+`;
+
+exports[`range-1.js 1`] = `
+thisWontBeFormatted  ( 1  ,3)
+
+    thisWillBeFormatted    (2  ,3<|>,   )
+
+    thisWontBeFormatted  (2, 90  ,)
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+thisWontBeFormatted  ( 1  ,3)
+
+    thisWillBeFormatted(2, 3<|>);
+
+    thisWontBeFormatted  (2, 90  ,)
+    
+`;
+
+exports[`range-2.js 1`] = `
+thisWontBeFormatted  ( 1  ,3)
+
+    thisWillBeFormatted    (2  ,3,  <|> )
+
+    thisWontBeFormatted  (2, 90  ,)
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+thisWontBeFormatted  ( 1  ,3)
+
+    thisWillBeFormatted(2, 3<|>);
+
+    thisWontBeFormatted  (2, 90  ,)
+    
+`;
+
+exports[`range-3.js 1`] = `
+thisWontBeFormatted <|> ( 1  ,3)
+
+    thisWillBeFormatted    (2  ,3,   )
+
+    thisWontBeFormatted  (2, 90  ,)
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+thisWontBeFormatted <|> ( 1  ,3)
+
+    thisWillBeFormatted(2, 3);
+
+    thisWontBeFormatted  (2, 90  ,)
+    
+`;
+
+exports[`range-4.js 1`] = `
+thisWontBeFormatted  ( 1  ,3)
+
+    thisWillBeFormatted    (2  ,3,   )
+
+    thisWontBeFormatted  (2, 9<|>0  ,)
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+thisWontBeFormatted  ( 1  ,3)
+
+    thisWillBeFormatted(2, 3);
+
+    thisWontBeFormatted  (2, 9<|>0  ,)
```

Comment:
Those snapshot formats are really cool, super easy to understand what’s going on!